### PR TITLE
Major Changes - Cylc Install - Rose Stem

### PR DESCRIPTION
**Pull request by wxtim**
Partial answer to https://github.com/cylc/cylc-doc/issues/170 Twinned with https://github.com/cylc/cylc-doc/pull/273

- Flesh out description of new `rose-stem` a little.
- Update description of Cylc Rose and list of Cylc utils where it is plumbed in.

### CODE REVIEW

File: cylc/rose/__init__.py [hunk 1]
```diff
@@ -113,12 +128,34 @@
...
+.. _rose-stem:
+
 Rose Stem
 =========
 
+.. seealso::
+
+   `Rose Stem documentation
+   <https://metomi.github.io/rose/2019.01.2/html/tutorial/rose/furthertopics/
+   rose-stem.html>`_
```
Comment (kinow) [inline cylc/rose/__init__.py:140]: I didn't know we could break lines in links this this. #TIL

**Review by kinow — APPROVED** [review]
👍

File: cylc/rose/__init__.py (hunk 1 above)
Comment (wxtim) [reply cylc/rose/__init__.py:140]: I didn't either, 'till I tried it!

File: cylc/rose/__init__.py [hunk 2]
```diff
@@ -113,12 +133,34 @@
...
+
+
 Cylc Rose provides a Rose Stem command, if FCM is installed on your system.
-See `the Rose Stem documentation
-<https://metomi.github.io/rose/2019.01.2/html/tutorial/rose/furthertopics/rose-stem.html>`_
+At Cylc 8 Rose Stem is a wrapper to ``cylc install`` rather than
+``rose suite-run``.
+
+Rose Stem is a wrapper around the ``cylc install`` command which
+provides some additional jinja2 variables.
```
Comment (datamel) [inline cylc/rose/__init__.py:153]:
```suggestion
provides some additional Jinja2 variables.
```

**Review by datamel — APPROVED** [review]
Thanks @wxtim, built along with its sibling. Looks good to me.

**Pull request merged**